tools/fixup_stuff.sh

```diff
@@ -202,5 +202,5 @@ fi
 # on python-virtualenv), first install the distro python-virtualenv
 # to satisfy any dependencies then use pip to overwrite it.
 
-install_package python-virtualenv
+install_package virtualenv-python
```

Comment:
I have a feeling that these may not be needed any more for anyone, but for now you could just check ! is_clearlinux here and skip it.  Does that work?